modules/microshift-providing-sos-report-support.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+
+// * microshift_support/microshift-sos-report
+
+:_content-type: PROCEDURE
+[id="providing-microshift-sos-report-support_{context}"]
+= Providing a {product-title} sos report to Red Hat support
```

Comment:
This information might be redundant, since in microshift_support/microshift-sos-report.adoc you link to the support procedure. Maybe we should just leave the link, in case the process changes or the support team wants to add details to the process?

Reply:
Let's delete this section completely.